documentation/AuthServer.md

```diff
@@ -2,20 +2,48 @@
 
 The authorization server adds access control to the cloud-based flow manger to secure access to user preferences. We are implementing the authorization server using the [OAuth 2.0 Authorization Framework](http://oauth.net/2/). See [GPII OAuth 2 Guide](https://wiki.gpii.net/w/GPII_OAuth_2_Guide) for the implementation of GPII supported OAuth 2.0 grant types.
 
+### Supported GPII Clients
+The authorization server authorizes 4 type of GPII clients:
+
+* GPII App installations
+    * For this client type, the local flow manager is the GPII component within a GPII App installation to communicate with GPII Cloud.

```

Comment:
Maybe something like: "The Local Flow Manger within the GPII App retrieves user settings from the GPII Cloud-Based Flow Manager."